src/llscan.cc

```diff
@@ -0,0 +1,470 @@
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <inttypes.h>
+
+#include <algorithm>
+#include <fstream>
+#include <vector>
+
+#include <lldb/API/SBExpressionOptions.h>
+
+#include "src/llnode.h"
+#include "src/llv8.h"
+#include "src/llv8-inl.h"
+#include "src/llscan.h"
+
+namespace llnode {
+
+// Defined in llnode.cc
+extern v8::LLV8 llv8;
+
+struct MemoryRange {
+    uint64_t start;
+    uint64_t length;
+    MemoryRange* next;
+};
+
+static MemoryRange* ranges = nullptr;
+
+std::map<uint64_t, TypeRecord*>* mapstoinstances = NULL;
```

Comment:
I suggest that we should use a `LLScan` class where `mapstoinstances` is a property. Command handling code could instantiate the class and pass it over to this file. Does it sound reasonable?